crypto/x509/x_attrib.c

```diff
@@ -178,24 +178,27 @@ int ossl_print_attribute_value(BIO *out,
         case NID_associatedName:
         case NID_dITRedirect:
         case NID_owner:
-            /*
-             * d2i_ functions increment the ppin pointer. See doc/man3/d2i_X509.pod.
-             * This preserves the original  pointer. We don't want to corrupt this
-             * value.
-             */
-            value = av->value.sequence->data;
-            xn = d2i_X509_NAME(NULL,
-                               (const unsigned char **)&value,
-                               av->value.sequence->length);
-            if (xn == NULL) {
-                BIO_puts(out, "(COULD NOT DECODE DISTINGUISHED NAME)\n");
-                return 0;
+            {
```

Comment:
How about this style?
```
            {
                /*
                * d2i_ functions increment the ppin pointer. See doc/man3/d2i_X509.pod.
                * This preserves the original  pointer. We don't want to corrupt this
                * value.
                */
                value = av->value.sequence->data;
                xn = d2i_X509_NAME(NULL,
                                (const unsigned char **)&value,
                                av->value.sequence->length);
                if (xn == NULL) {
                    BIO_puts(out, "(COULD NOT DECODE DISTINGUISHED NAME)\n");
                    return 0;
                }

                int wrote = X509_NAME_print_ex(out, xn, indent, XN_FLAG_SEP_CPLUS_SPC);
                X509_NAME_free(xn);
                return wrote > 0 ? 1 : 0;    
            }
```

Reply:
This project only does variable definitions at the start of blocks.

Reply:
Code review note not related to these changes, but noticed during my due-dilligence into what `X509_NAME_print_ex` returns, entered in Issue https://github.com/openssl/openssl/issues/25209

Reply:
I would just declare the ret variable at the beginning of the function instead of adding a block here.